### PR TITLE
[KHWEBA-7] add account dialog

### PR DESCRIPTION
**Pull request by gift-a**
image

### CODE REVIEW

**Comment by ihor7016** [comment]
please use input without label but with placeholder and hint underneath (like in transactions dialog)

**Comment by ihor7016** [comment]
image

we need all these fields (name, type and currency) in our dialog (except sync with bunk link)

**Comment by ihor7016** [comment]
BTW buttons should be Cancel and Ok (please also rename methods in js according to this changes)

File: src/index.scss [hunk 1]
```diff
@@ -4,3 +4,4 @@
 
 @import "./components/page/page";
 @import "./components/app/app";
+@import "./components/add-account-dialog/add-account-dialog";
```
Comment (ihor7016) [inline src/index.scss]: please move this import to html

**Review by ihor7016 — DISMISSED** [review]
~~will be merged after the drawer~~
UPD -- please fix webpack and importants !!! see https://github.com/ihor7016/buxy-vanilla/pull/2#pullrequestreview-102879082

File: src/index.scss (hunk 1 above)
Comment (gift-a) [reply src/index.scss]: what do you mean? "to html"

**Comment by gift-a** [comment]
have a doubt about how to mount and subscribe this component. have 2 variants:
1. import and mount in app, subscribe in drawer. In this case I need to send the action to dialog via app using props.
~~2. import, mount and subscribe on event in drawer (don't like it. I think dialog have to be in container)~~
~~3. import and mount in app, subscribe in app on link `<a class="drawer__add-account-dialog-activation">` or `<a class="app__add-account-dialog-activation">`? (don't like it too)~~

**Comment by gift-a** [comment]
done

File: src/components/drawer/drawer.js [hunk 2]
```diff
@@ -2,14 +2,29 @@ import template from "./drawer.html";
...
 export class DrawerComponent {
-  constructor(mountPoint) {
+  constructor(mountPoint, props) {
     this.mountPoint = mountPoint;
+    this.props = props;
   }
 
   querySelectors() {
     this.drawerRoot = this.mountPoint.querySelector(".mdc-drawer--persistent");
     this.drawer = new MDCPersistentDrawer(this.drawerRoot);
```
Comment (ihor7016) [inline src/components/drawer/drawer.js]: please move this to `initMDC()` method

File: src/components/app/app.js [hunk 3]
```diff
@@ -50,6 +58,10 @@ export class AppComponent {
     this.addTransacionDialog.showDialog();
```
Comment (ihor7016) [inline src/components/app/app.js]: typo here `addTransacionDialog ` --> `addTransactionDialog` (`t` letter)

**Comment by gift-a** [comment]
ready

**Review by ihor7016 — APPROVED** [review]

**Pull request merged**